scripts/update-weights-setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -exu
+
+# Set up the standardized machine and run `update-weights.sh` script.
+# The system is assumed to be pristine Ubuntu 20.04 and we install
+# all required dependencies.
```

Comment:
We should probably note down the expected machine specs somewhere, but not here since this script doesn't care as long as it's Ubuntu

Reply:
We have it on our internal notion page (Deployments). I wasn't sure if we should advertise a specific provider :)

Reply:
We can list specs without listing a provider (not that it matters imo). It would be useful for people who need a reference for their own Substrate based chains.

Just a thought though, this can be merged without that.